.gitignore

```diff
@@ -1,6 +1,7 @@
 #  SR User Related   #
 ######################
 cache*/
+data*/
 Logs/
```

Comment:
Default logdir is inside DATA_DIR, so it is already covered by the gitignore of data/
This doesn't need changed